src/mercury_bulk.c

```diff
@@ -1779,6 +1789,37 @@ HG_Bulk_transfer(hg_context_t *context, hg_cb_t callback, void *arg,
         origin_handle, origin_offset, local_handle, local_offset, size, op_id);
 }
 
+/*---------------------------------------------------------------------------*/
+hg_return_t
+HG_Bulk_bind_transfer(hg_context_t *context, hg_cb_t callback, void *arg,
+    hg_bulk_op_t op, hg_bulk_t origin_handle, hg_size_t origin_offset,
+    hg_bulk_t local_handle, hg_size_t local_offset, hg_size_t size,
+    hg_op_id_t *op_id)
```

Comment:
not sure why need to use another different API? 
it seems that it is easier for user to use if can reuse the same API and differentiate the internal handling by some internal info?

Reply:
Well I would prefer for now that there is a separate API call, it does not seem great otherwise if you have to pass either an addr that is not going to be used or a NULL_ADDR